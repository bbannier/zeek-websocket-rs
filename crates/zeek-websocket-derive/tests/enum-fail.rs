use zeek_websocket_derive::ZeekType;

#[derive(ZeekType)]
enum E {
    A,
    B,
    C,
}

fn main() {}
