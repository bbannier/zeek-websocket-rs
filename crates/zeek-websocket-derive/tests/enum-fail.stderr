error: only structs can derive ZeekType
 --> tests/enum-fail.rs:4:1
  |
4 | enum E {
  | ^^^^
